packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -237,19 +253,22 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
     // During mounting we don't know the child context yet as the instance doesn't exist.
     // We will invalidate the child context in finishClassComponent() right after rendering.
     const hasContext = pushLegacyContextProvider(workInProgress);
+    const instance = workInProgress.stateNode;
```

Comment:
You're not using the instance. Let's inline it back in place. It avoids reading the stateNode property here when current is not null.